modules/openapi-generator/src/main/resources/codegen/generatorClass.mustache

```diff
@@ -110,8 +110,7 @@ public class {{generatorClass}} extends DefaultCodegen implements CodegenConfig
 
     /**
      * Reserved words.  Override this with reserved words specific to your language
-     */
-    reservedWords = new HashSet<String> (
+     */ super.registerReservedWordsCaseSensitive(
```

Comment:
add newline

Reply:
Resolved in commit [0860bd06](https://github.com/OpenAPITools/openapi-generator/commit/0860bd06dd94abff420fafac93ffb7a12607fef1)